apis/secretmanager/v1beta1/secret_types.go

```diff
@@ -48,8 +48,7 @@ type SecretManagerSecretSpec struct {
 	//  This is always provided on output, regardless of what was sent on input.
 	ExpireTime *string `json:"expireTime,omitempty"`
 
-	// Input only. The TTL for the
-	//  [Secret][google.cloud.secretmanager.v1.Secret].
+	// Input only. A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
```

Comment:
The TTL is a stale field that has causes issues (it uses "Now()" when the resource is created that users cannot control or predict)

Reply:
Ouch.  This doesn't feel like a spec field, because if I specify `ttl: 60s` what is the TTL value in 10 seconds?  It seems like it should be 60s, because that is my desired state.  But I suspect it will be 50s.

Reply:
Yes, this is exactly the problem!